persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -82,6 +86,14 @@ public void execute() {
             errStream.println(commandUsageInfo);
             return;
         }
+
+        if (datastore == null) {
+            datastore = KEYWORD_MYSQL;
+        } else if (!datastore.equals(KEYWORD_MYSQL)) {
+            errStream.println("ERROR: only 'mysql' is supported for the datastore.");
```

Comment:
```suggestion
            errStream.println(String.format("ERROR: the persist layer supports only 'mysql' datastore. but found '%s' datasource.", datastore));
```

Reply:
We need to check this in generate command as well. read Ballerina.toml file and validate datasource

Reply:
added in [566e36d](https://github.com/ballerina-platform/persist-tools/pull/148/commits/566e36dd84d453026fbf2506266f34668c08e1af)